release-checklist.md

```diff
@@ -0,0 +1,21 @@
+# Release checklist
+- [ ] `git flow release start v19.3.0` (`v19.3.0-beta.1` if beta)
```

Comment:
- [ ]  `git flow release start VERSION_NUMBER` 
   Version numbers start with `v` and we support `beta` releases.     For example, `19.3.0` or 19.3.0-beta.1` are both valid `VERSION_NUMBER` values.

Reply:
Addressed in update